data-pipeline/src/pipeline/rag.py

```diff
@@ -121,23 +121,32 @@ def category_stats(urn, category_name, data, ofsted_rating, rag_mapping, close_c
 
     percentile = find_percentile(series, value)
     decile = int(percentile / 10)
-    mean = np.median(series)
-    diff = value - mean
+    mean = np.mean(series)
+    diff_mean = value - mean
     diff_percent = (
-        (diff / mean) * 100
+        (diff_mean / mean) * 100
         if mean != 0 and mean != np.inf and mean != np.nan and not pd.isna(mean)
         else 0
     )
+    median = np.median(series)
+    diff_median = value - median
+    diff_percent = (
```

Comment:
`diff_percent` has me a little confused. Seems like it is being assigned based on the mean and then again based on the median.

Feels like we either need both diff_percent_mean & diff_percent_median, or just one which is based on either mean or median?

Reply:
I added a comment a little further down to say that value wasn't being consumed anywhere on the front end, so this distinction was not of concern. Happy to add another field if you feel it is necessary to persist this value.